src/dynatrace/opentelemetry/metrics/export/_factory.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2020 Dynatrace LLC
```

Comment:
```suggestion
# Copyright 2022 Dynatrace LLC
```

Reply:
fixed in [0ce3ad9](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/27/commits/0ce3ad925520917615a5767d9205f81dda0762a5) :slightly_smiling_face: